site/content/get-started.md

```diff
@@ -305,7 +305,8 @@ Request ID: 2dfc85564dd5b5dad7e62b980bb60ee1
 
 ---
 
-## Additional reading
+## See also
 
-- [Install NGINX Gateway Fabric](http://localhost:1313/nginx-gateway-fabric/installation/installing-ngf/), for additional ways to install NGINX Gateway Fabric
-- [How-to guides](http://localhost:1313/nginx-gateway-fabric/how-to/), for configuring your cluster
+- [Install NGINX Gateway Fabric]({{< ref "/installation/installing-ngf/" >}}), for additional ways to install NGINX Gateway Fabric
+- [How-to guides]({{< ref "/how-to/" >}}), for configuring your cluster
+- [Traffic management]({{< ref "/how-to/traffic-management/" >}}), for more in-depth traffic management configuration
```

Comment:
Do these need punctuation at the end?

Reply:
Technically no, but I don't think we've ever had a strict rule about this. I consider bullet points to have implicit punctuation due to line breaks.

I didn't add punctuation to the prerequisites list either: it's internally consistent within the single document, but I haven't audited totality of documentation.